Extensions/IISWebAppDeploy/Src/Tasks/SqlDacpacDeploy/DeployToSqlServer.ps1

```diff
@@ -81,22 +75,73 @@ function GetScriptToRun
 
     if ($taskType -eq "dacpac")
     {
-        $invokeMain = "Execute-DacpacDeployment -dacpacFile `"$dacpacFile`" -targetMethod $targetMethod -serverName `"$serverName`" -databaseName `"$databaseName`" -authscheme $authscheme -sqlServerCredentials `$sqlServerCredentials -connectionString `"$connectionString`" -publishProfile `"$publishProfile`" -additionalArguments `"$additionalArguments`""
-
+        
         $sqlPackageScript = Get-Content TaskModuleSqlUtility\SqlPackageOnTargetMachines.ps1 | Out-String
 
+        $sqlSplatArguments = @{
+            dacpacFile=$dacpacFile 
+            targetMethod=$targetMethod 
+            serverName=$serverName
+            databaseName=$databaseName
+            authscheme=$authscheme 
+            sqlServerCredentials="`$sqlServerCredentials"
+            connectionString=$connectionString 
+            publishProfile=$publishProfile 
+            additionalArguments=$additionalArguments
+        }
+        $jsonSqlSplatArguments = $sqlSplatArguments | ConvertTo-Json
```

Comment:
will be good if we can place this in try catch and print the error in catch block